command/shell.go

```diff
@@ -142,6 +142,9 @@ func (c *shellExec) Execute(ctx context.Context,
 		env = append(env, fmt.Sprintf("%s=%s", subprocess.MarkerTaskID, conf.Task.Id),
 			fmt.Sprintf("%s=%d", subprocess.MarkerAgentPID, os.Getpid()))
 		localCmd.Environment = env
+		if ctx.Err() != nil {
```

Comment:
Is this just going to handle canceling the context or could there be some error that we would want to return?

Reply:
We return an error on 202.